### DISCUSSION TITLE
Filament 3.2 support

### DISCUSSION THREAD

**Discussion by mebron** [chiiya/filament-access-control#54] [Q&A]
This is a very usable and must-have plugin. It says it requires filament ^3.0 but it is not supporting 3.2 right now. When trying to install I have getting following error:

![image](https://github.com/chiiya/filament-access-control/assets/872739/a8ce79d6-3048-445e-9754-3df97a16c4b7)

**Comment by chiiya** [comment]
I don't think this is an issue with Filament 3.2, but rather with your dependency on spatie/laravel-permission. Apparently in your composer.json you require spatie/laravel-permission ^6.3, whereas this package is not compatible with 6+ yet. It should work if you remove that dependency from your composer.json.
Aside from that, I should also make sure this package is compatible with spatie/laravel-permission ^6.0.

**Reply by mebron** [reply]
Thank you for pointing out the dependency issue with spatie/laravel-permission in our composer.json. We will reevaluate our requirement for spatie/laravel-permission ^6.3 and make the necessary adjustments to ensure compatibility. In the meantime, removing this dependency as you suggested could be a temporary solution. Additionally, we will prioritize updating our package to be compatible with spatie/laravel-permission ^6.0 to prevent similar issues in the future. Your feedback is invaluable in helping us maintain and improve our software's reliability
